optimizer-fix-controls.md

```diff
@@ -71,6 +71,14 @@ SET SESSION tidb_opt_fix_control = '44262:ON,44389:ON';
 - This variable sets the threshold for the optimizer's heuristic strategy to select access paths. If the estimated rows for an access path (such as `Index_A`) is much smaller than that of other access paths (default `1000` times), the optimizer skips the cost comparison and directly selects `Index_A`.
 - `0` means to disable this heuristic strategy.
 
+### [`47400`](https://github.com/pingcap/tidb/issues/47400) <span class="version-mark">New in v8.4.0</span>
+
+- Default value: `ON`
+- Possible values: `ON`, `OFF`
+- Due to challenges in accurately estimating the number of qualified rows for each plan step in a query plan, the optimizer often sets a smaller value for `estRows`. This variable controls whether to limit this minimum value.
+- `ON`: sets the minimum value of `estRows` to 1, which is the new behavior introduced in v8.4.0 and is consistent with other databases.
```

Comment:
```suggestion
- `ON`: limits the minimum value of `estRows` to 1, which is the new behavior introduced in v8.4.0 and is consistent with other databases.
```

Reply:
"other databases" seems ambiguous. Does "other databases" include MySQL? If yes, it is better to use "MySQL" to replace "other databases".

Reply:
"Other" refers to Oracle and Db2. I am not sure what MySQL does.